services/motion/localizer.go

```diff
@@ -66,20 +125,29 @@ func (m *movementSensorLocalizer) CurrentPosition(ctx context.Context) (*referen
 	if err != nil {
 		return nil, err
 	}
+
+	if err := validateGeopoint(gp); err != nil {
```

Comment:
Instead of this happening here I think this should just happen when we convert the protobuf to an RDK geopoint